.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@4.35.6
+
+workflows:
+  package-and-push-chart-on-tag:
+    jobs:
+      - architect/push-to-registries:
+          context: architect
+          name: push-to-registries
+          push-to-quay: true
+          push-to-docker: false
+          push-to-gsoci: false
+          push-to-aliyun: false
```

Comment:
hmm do we need all for this tool? 🤔

Reply:
`docker.io` may make sense, but since the private Intranet images are also only hosted on `quay.io`, we may limit this one just as well.

In the `cms/Dockerfile.cms` (for local development or docs writing) I specified the image including registry (fully qualified).